src/java/main/org/apache/jute/Utils.java

```diff
@@ -190,19 +190,32 @@ static String fromCSVString(String s) throws IOException {
     }
     
     /**
+     * convert byte array to a string in hex format
      * 
-     * @param s 
-     * @return 
+     * @param barr
+     * @return
      */
-    static String toXMLBuffer(byte barr[]) {
```

Comment:
I'm not able to dig into Zookeeper internal to find the root cause.
but this link would help:
http://stackoverflow.com/questions/12498256/why-the-npe-using-static-method-of-datatypeconverter
https://developer.ibm.com/answers/questions/200553/jaxb-datatypeconverter-throws-nullpointerexception.html